src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml.cpp

```diff
@@ -66,6 +66,17 @@ GraphingCalculator::GraphingCalculator()
 
     // And when the actual trace value changes
     GraphingControl->TracingValueChangedEvent += ref new TracingValueChangedEventHandler(this, &GraphingCalculator::OnTracePointChanged);
+
+    // OemMinus and OemAdd aren't declared in the VirtualKey enum, we can't add this accelerator XAML-side
+    auto virtualKey = ref new KeyboardAccelerator();
+    virtualKey->Key = (VirtualKey)187; //OemMinus key
+    virtualKey->Modifiers = VirtualKeyModifiers::Control;
```

Comment:
Why the need for Ctrl instead of just +/- directly? I think it might be more intuitive and discoverable.

Reply:
Specs :D

I suppose to have the same behavior than Maps.

cc @grochocki